cmdstanpy/install_cmdstan.py

```diff
@@ -337,12 +303,25 @@ def retrieve_version(version: str, progress: bool = True) -> None:
             ) from e
     print('Download successful, file: {}'.format(file_tmp))
     try:
+        print('Extracting distribution')
         tar = tarfile.open(file_tmp)
         target = os.getcwd()
         if platform.system() == 'Windows':
             # fixes long-path limitation on Windows
             target = r'\\?\{}'.format(target)
-        tar.extractall(target)
+
+        if progress:
```

Comment:
This currently does nothing for me

Reply:
what do you mean?  how does Windows handle unpacking a tgz file?

Reply:
I'm running on ubuntu at the moment and 

```python
>>> import cmdstanpy
>>> cmdstanpy.install_cmdstan("2.26.0", progress=True)
```

displays no progress bars during download or unpacking

Reply:
it works on a Mac (Catalina),  but only after:
 - installing develop version as latest version for system library
 - calling the python script from the comnand line.

but if this is too limited, OK with getting rid of it.

Reply:
I was referring to the progress bar not the tar unpacking, may have been unclear. I think this is covered by #462